guides/REAME.md

```diff
@@ -0,0 +1,9 @@
+# Guides
+
+This folder contains documents for how to setup the CI/CD flow for a Knative
+project with the existing tools/scripts provided in this repository.
+
+- [How to setup Prow](./prow_setup.md)
+- [How to setup CI/CD flows for a new repo with helper scripts](../scripts)
```

Comment:
Isn't this for how to write test scripts?

Reply:
Changed to `How to implement your own test/release scripts with the helper scripts`